README.md

```diff
@@ -136,6 +136,65 @@ const response = await signIn({
 });
 ```
 
+### Next.js (Pages Router) with OpenID Connect
+
+Setup `_app.tsx` as described above. In your login component trigger the external auth flow using the following code:
+
+```tsx
+import { Inter } from 'next/font/google'
+
+import { useSaleorAuthContext, useSaleorExternalAuth } from '@saleor/auth-sdk/react'
+import { ExternalProvider } from '@saleor/auth-sdk';
+import Link from 'next/link';
+import { gql, useQuery } from '@apollo/client';
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function Home() {
+  const { loading: isLoading, error, data } = useQuery(gql`query CurrentUser { me { id email firstName lastName } }`);
+
+  const { authURL, loading } = useSaleorExternalAuth({
+    saleorURL: '<your Saleor instance>',
+    provider: ExternalProvider.OpenIDConnect,
+    redirectURL: 'http://localhost:5375/api/auth/callback',
+  });
+
+  const { signOut } = useSaleorAuthContext();
+
+  if (loading || isLoading) {
+    return <div>Loading...</div>;
+  } else if (data && data.me) {
+    return (
+      <div>
+        {JSON.stringify(data)}
+        <button onClick={() => signOut()}>Logout</button>
+      </div>
+    )
+  } else if (authURL) {
+      return (
+      <div>
+        <Link href={authURL}>Login</Link> 
+      </div>
+      )
+  } else {
+    return (
+      <div>Something went wrong</div>
+    )
+  }
+}
```

Comment:
```suggestion
import {
  useSaleorAuthContext,
  useSaleorExternalAuth,
} from "@saleor/auth-sdk/react";
import { ExternalProvider } from "@saleor/auth-sdk";
import Link from "next/link";
import { gql, useQuery } from "@apollo/client";

export default function Home() {
  const {
    loading: isLoadingCurrentUser,
    error,
    data,
  } = useQuery(
    gql`
      query CurrentUser {
        me {
          id
          email
          firstName
          lastName
        }
      }
    `
  );
  const { authURL, loading: isLoadingExternalAuth } = useSaleorExternalAuth({
    saleorURL: "<your Saleor instance>",
    provider: ExternalProvider.OpenIDConnect,
    redirectURL: "http://localhost:5375/api/auth/callback",
  });

  const { signOut } = useSaleorAuthContext();

  if (isLoadingExternalAuth || isLoadingCurrentUser) {
    return <div>Loading...</div>;
  }
  if (data?.me) {
    return (
      <div>
        {JSON.stringify(data)}
        <button onClick={() => signOut()}>Logout</button>
      </div>
    );
  }
  if (authURL) {
    return (
      <div>
        <Link href={authURL}>Login</Link>
      </div>
    );
  }
  return <div>Something went wrong</div>;
}
```

Reply:
(formatted code)

Second question: is http://localhost:5375 correct?

Reply:
@mmiszy I've clarified the `localhost`, but this will be changed once we have the 1st version out